### DISCUSSION TITLE
Project Logo seems broken

### DISCUSSION THREAD

**Discussion by MucahitErdoganUnlu** [bounswe/bounswe2024group11#157] [General]
![image](https://github.com/bounswe/bounswe2024group11/assets/94074126/ab5f5465-6a7c-4013-831d-8be55ce815b1)
Our logo seems broken. Maybe something is wrong on the github side and not on us?

**Comment by MucahitErdoganUnlu** [comment]
@umitcan07

**Comment by umitcan07** [comment]
Must be github, turns on and off from time to time

**Comment by Meminseeker** [comment]
I'm also experiencing this issue for a while

**Comment by MucahitErdoganUnlu** [comment]
Thank you [Ümit](https://github.com/bounswe/bounswe2024group11/discussions/157#discussioncomment-9146543)! Closing it.
